File: ARCADE/The Core/Level 4_6: appleBoxes.cs
    using System;
     
    public class Test
    {
    	public static int appleBoxes(int k){
            int yellow = 0, red = 0;
            for (int i = 1; i <= k; i++){
                if (i % 2 == 0) red = red + i * i;
                else yellow = yellow + i * i;
            }
            return red - yellow;
        }
    	public static void Main()
    	{
    		int k = 5;
    		Console.Write(appleBoxes(k));
    	}
    }

File: ARCADE/The Core/Level 4_6: appleBoxes.pas
program appleBoxes;
 
function appleBoxes(k: integer): integer;
var yellow, red, i : integer;
begin
    yellow := 0; red := 0;
    for i := 1 to k do
    begin
        if(i mod 2 = 0) then inc(red, i * i)
        else inc(yellow, i * i);
    end;
 
    appleBoxes := red - yellow;
end;
 
var k : integer;
begin
	k := 5;
	writeln(appleBoxes(k));
end.
